tests/drum/test_custom_model.py

```diff
@@ -90,8 +90,7 @@ def returncode(self):
 class DrumServerRun:
```

Comment:
`DrumServerRun` is initialized with various args, I don't see how and why we would convert it to fixture.
let's keep it as is, but move to `utils.py`